anchor-contributor/programs/anchor-contributor/src/lib.rs

```diff
@@ -73,6 +73,32 @@ pub mod anchor_contributor {
             ContributorError::InvalidVaaPayload
         );
 
+        // We need to verify that the accepted tokens are actual mints.
+        let assets = &sale.totals;
+        let accepted_mints = &ctx.remaining_accounts[..];
+        require!(
+            assets.len() == accepted_mints.len(),
+            ContributorError::InvalidRemainingAccounts
+        );
```

Comment:
Passing each accepted mint into remaining accounts to verify that these tokens exist